### PR TITLE
Slava110

### PR DESCRIPTION
**Pull request by dmodoomsirius**

### CODE REVIEW

**Review by dmodoomsirius — COMMENTED** [review]
I'm good with the gradle changes.

**Comment by voidsong-dragonfly** [comment]
Doubt anything here will affect stuff because basically no one is building for 1.12, but I'd like to get @zmaster587 's input as well

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    if (bambooshortPlanName == "BBM-Dev")
+        archiveClassifier.set("deobf-dev")
+    else
+        archiveClassifier.set("deobf")
+    //sets where the jars go to.
+    destinationDirectory.set(file("output"))
+}
+
+tasks.jar {
+    //if project has the property BBM-Dev then it appends dev to indicate the dev branch
```
Comment (dmodoomsirius) [inline build.gradle.kts]: this whole jar task can be removed. I thought about it and i don't see a need for it anymore.

Comment (slava110) [reply build.gradle.kts]: Removed

**Comment by slava110** [comment]
> Doubt anything here will affect stuff because basically no one is building for 1.12, but I'd like to get @zmaster587 's input as well

That's up to you if you want to use this for 1.12 but... You can use it for 1.16 aswell actually. Because it's already using ForgeGradle 5 you just need to do some minor changes for it to work (change version number and remove FancyGradle stuff)

**Comment by slava110** [comment]
Also it's possible to rewrite it in groovy if you want (very similar to Kotlin DSL)
But there's a problem with artifacts trashing output directory (libVulpes jar will be created before each launch). Possible fixes: remove jar or run clean task before launch
And I think build number should not be included in mod version
UPD: Also these changes will actually require changes in libVulpes too (so run configurations will work properly)

**Pull request closed without merging**